src/github/filterStaleBranches.js

```diff
@@ -1,12 +1,13 @@
 const { github } = require('../config');
 
-module.exports = ({ nodes, excludedBranches = github.exclude }) => {
+// filters stale, default and excluded branches
+module.exports = ({ nodes, defaultBranchName = null, excludedBranches = github.exclude }) => {
   const past = new Date();
   past.setMonth(past.getMonth() - 3);
 
   const staleBranches = nodes
     .filter(({ target: { committedDate } }) => Date.parse(committedDate) < past)
-    .filter(({ name }) => !excludedBranches.includes(name));
+    .filter(({ name }) => ![...excludedBranches, defaultBranchName].filter(Boolean).includes(name));
```

Comment:
Considering we're going to provide more ways to exclude branches going forward, I would like to discuss how to accommodate these changes making this part of code still simple.
Off the top of my head, we could extract the logic for making a list of branches that need to be excluded into a separate module.